packages/api/src/hooks/p2p/useCreateAdvertiser.ts

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useMemo } from 'react';
+import useAuthorize from '../useAuthorize';
+import useMutation from '../../useMutation';
+import useInvalidateQuery from '../../useInvalidateQuery';
+
+type TAdvertiseCreatePayload = {
+    name: string;
+};
+
+const useCreateAdvertiser = () => {
+    const invalidate = useInvalidateQuery();
+    const { isSuccess: isUserAuthorized } = useAuthorize();
+    const { data, mutate, ...rest } = useMutation('p2p_advertiser_create', {
+        onSuccess: () => {
+            invalidate('p2p_advertiser_list');
+            invalidate('p2p_advertiser_info');
```

Comment:
not sure about invalidating `p2p_advertiser_list`

Reply:
i think its fine to invalidate it, since the list needs to be updated with your information as well 👍

Reply:
ok then. will keep it. thanks @ameerul-deriv

Reply:
sorry i just edited my comment 😆 can you take a look at which one api endpoint you want to invalidate? is it `p2p_advertiser_adverts` or `p2p_advert_list`

Reply:
oops. I thought we have an endpoint to fetch all advertisers as well.  so the only one that is need to be invalidate is just `p2p_advertiser_info`

Reply:
the type is wrong!
that's why I added `p2p_advertiser_list`

![image](https://github.com/binary-com/deriv-app/assets/100833613/a94795b3-7384-4268-8c7a-e7926f6efea2)

Reply:
ahhhh alright 👍 yeah we dont have that endpoint 😓

Reply:
I dont think we should invalidate this endpoint, p2p_advertiser_list is the list of advertisers that has trade with the current advertiser (which in this case none if we just created new account) 🤔

Reply:
I will remove it . later if we need this we can add it.